src/registry_schemas/schemas/agm_extension.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/agm_extension",
+    "type": "object",
+    "definitions": {},
+    "title": "AGM Extension Information Schema",
+    "required": [
+        "agmExtension"
+    ],
+    "properties": {
+        "agmLocationChange": {
+            "type": "object",
+            "required": [
+                "year"
+            ],
+            "properties": {
+                "year": {
+                    "type": "string",

```

Comment:
Would it make sense to change this to "integer"?

Reply:
if you do a number type, you'll need a min/max, or stay with string and provide a regex (4 chars only and appropriate ranges)

Reply:
Is there an appropriate range for AGM Year?

Reply:
well, shouldn't be an unbounded integer eg 26,000 is a valid year, but just one I'm unlikely to see.

Reply:
also sounds like it will require a coded validator (if 6 months is the maximum extension, it should probably be the date is was extended to)

Reply:
Yes, there will be a data field that needs to be validated down to the month, but it's not AGM Year.

Hmm, maybe we should keep this as a string, same as AGM Location Change, or change that one to an integer as well? Check with whoever is working with that one.https://github.com/bcgov/business-schemas/blob/3b55629ff95f3a2e857ca0774f14b46fcceec82c/src/registry_schemas/schemas/agm_location_change.json#L18

Reply:
> also sounds like it will require a coded validator

Yes, that will be done in [this ticket](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/18053).

Reply:
OK, so it stays as a string 👍